styles/styles.css

```diff
@@ -575,6 +674,39 @@
 
 }
 
+@media (max-width: 740px) {
+  .villa {
+    display: initial;
+  }
+
+  .content__guests--home {
+    width: auto;
+    height: 193px;
+  }
+
+  .content__guests--title {
+    font-size: 28px;
+    line-height: 33px;
+    padding-bottom: 32px;
+  }
+
+  .content__guests--home-img {
```

Comment:
Your approach not bad in this case, however I think, that it's better to make ot more scalable. Use instade of fixed with and height 100% , and use max-width: 155px and max-height:155px

Reply:
what do you mean by fixed width? i thought that fixed means some defined value, like 155px

Reply:
Can you please explain this point and review the rest, i just want it to be merged to be able to proceed with other tasks.